stratum/hal/lib/tdi/tdi_sde_counter.cc

```diff
@@ -57,33 +57,28 @@ ::util::Status TdiSdeWrapper::WriteIndirectCounter(
   if (byte_count.has_value()) {
     tdi_id_t field_id;
     dataFieldInfo = table->tableInfoGet()->dataFieldGet(kCounterBytes);
-    RETURN_IF_NULL(dataFieldInfo);
-    field_id = dataFieldInfo->idGet();
-    RETURN_IF_TDI_ERROR(table_data->setValue(field_id, byte_count.value()));
+    if (dataFieldInfo) {
```

Comment:
This code is doing the same as old code. Old code returns if null and new code doesnt seem to do anything different. Am I missing something?

Reply:
There is a difference between Sabeel. Old code flags the error and returns from the processing.
Modified code will continue with further processing for available fields.